docs/source/hosted_applications/NREL_APPS/Real_Time_DER_Setpoint_Dispatch.rst

```diff
@@ -47,32 +47,33 @@ the updated model and measurements.
 its interaction with GridApps=D.**
 
 Data requirements
+~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
headers should only goto the end of the line

eg

Data requirements
~~~~~~~~~~~~

Reply:
Okay fixed.